src/ai/backend/manager/api/vfolder.py

```diff
@@ -3055,6 +3127,7 @@ def create_app(default_cors_options):
     cors.add(add_route("GET", r"/_/mounts", list_mounts))
     cors.add(add_route("POST", r"/_/mounts", mount_host))
     cors.add(add_route("DELETE", r"/_/mounts", umount_host))
+    cors.add(add_route("POST", r"/_/change_ownership", change_vfolder_ownership))
```

Comment:
Using underbar to represent whitespace is now considered as legacy naming convention. Please use hyphen.
```suggestion
    cors.add(add_route("POST", r"/_/change-ownership", change_vfolder_ownership))
```